More environment references (#7015)
umbraco/UmbracoDocs: umbraco-cloud/product-upgrades/version-specific-upgrades/migrate-from-8-to-latest.md

## Prerequisites

* A Umbraco 8 Cloud project running **the latest version of Umbraco 8**.
* A clean Cloud project running the latest version of Umbraco with **at least two environments**.
* A backup of your Umbraco 8 project database.

> We strongly recommend having at least **two environments** on the new Umbraco project.

Reviewer: ```suggestion
It is recommended to have at least **two environments** on the new Umbraco project.
```
